static/css/section/_desktop.scss

```diff
@@ -1,258 +1,153 @@
-      padding-bottom: 5px;
+      .pull-right {
+        max-width: 60%;
+      }
     }
   }
 
-  .eight-col,
-  .eight-col.box,
-  .six-col,
-  .nine-col {
-    padding: 20px;
-  }
-
-  .six-col,
-  .six-col,
-  .eight-col {
-    background:$white;
-    position: relative;
+  .landscape-screenshot {
+    -webkit-box-shadow: -2px 0px 2px 1px rgba(0, 0, 0, 0.1);
+    -moz-box-shadow: -2px 0px 2px 1px rgba(0, 0, 0, 0.1);
+    box-shadow: -2px 0px 2px 1px rgba(0, 0, 0, 0.1);
   }
 
-  .nine-col div,
-  .nine-col div {
-    background: #fff;
+  .share-this {
+    clear: both;
     display: block;
-    float: left;
-  }
-}
-
-.row-case-studies-light {
-  background-color: $light-grey;
-  background-image: none;
-  color: #333;
-
-  .pull-quote p,
-  .pull-quote cite {
-    color: #333;
-  }
-}
-
-@media only screen and (min-width: $breakpoint-medium) {
-  //Medium screen
-  .row-apps {
-    h3 {
-      margin-top: 5%;
-    }
-  }
+    height: 25px;
+    margin-top: 20px;
+    position: relative;
+    width: 100%;
 
-  .row-case-study {
-    .headline {
-      font-size: 1.7737857143em;
+    iframe {
+      display: inline-block;
+      float: left;
+      margin: 0;
+      padding: 0;
+      width: 110px;
     }
 
-    blockquote {
-      margin-bottom: 20px;
+    .twitter-share-button {
+      display: inline-block;
+      float: none;
+      margin-right: 18px;
+      width: 58px;
     }
   }
 }
 
+// @section desktop-home
+.desktop-home {
 
-/// desktop Overview
-.desktop-home #main-content {
-  .row-industry img {
-    display: block;
-    margin: 0 auto 10px;
-    max-width: 150px;
-  }
-
-  .row-latest-hardware .eight-col,
-  .row-tour .six-col {
-    margin-bottom: 20px;
-  }
+  .row-hero {
 
-  //Medium screen
-  @media only screen and (min-width: $breakpoint-medium) {
-    .row-hero {
+    @media only screen and (min-width: $breakpoint-medium) {
       background-color: $light-grey;
       background-image: url('#{$asset-server}364631d0-desktop-overview-hero.jpg?w=984');
-      background-repeat: no-repeat;
       background-position: 73% 13%, bottom right;
+      background-repeat: no-repeat;
+      border-bottom: 0;
       margin-top: 0;
       min-height: 573px;
-      border-bottom: 0;
 
       h1 {
         margin-top: 40px;
       }
 
       .light {
-        color: #333;
+        color: $cool-grey;
       }
     }
 
-    .row-latest-hardware .eight-col,
-    .row-tour .six-col {
-      margin-bottom: 0;
-    }
-
-    .row-industry .box {
-      overflow: hidden;
-      padding-bottom: 0;
-
-      img {
-        margin: 0;
-        margin-bottom: -60px;
-        padding-left: 40px;
-      }
-    }
-  }
-
-  @media only screen and (max-width: 844px) {
-    .row-complete .pull-right {
-      margin-top: 7%;
-    }
-  }
-
-  //Large screen
-  @media only screen and (min-width: $breakpoint-large) {
-    .row-hero {
+    @media only screen and (min-width: $breakpoint-large) {
       background-size: cover;
       margin-top: 0;
```

Comment:
This doesn't need repeating, already declared in `min-width: $breakpoint-medium`.